### ISSUE TITLE
App doesn't show response.

### ISSUE THREAD

**Issue by RNetc0rp** [t3mr0i/ChatGPTGlass#1]
Hello, I recently compiled your app and installed it on my Glass XE-C running the XE24 and the app works up to the point where it is supposed to provide a response.  It then just goes back to tap the touchpad.

**Comment by RetouchedDevelop** [comment]
@RNetc0rp & @t3mr0i Im pretty interested in... how you actually managed to get the app installed, when i try to build it in android studio i get the error

Error running App:
Default Activity not found

**Comment by RNetc0rp** [comment]
I ran into that issue as well.  For me I found installing an older version of android studio and going through the Glass Development Kit installation again fixed that issue.

**Comment by rpcribari** [comment]
Did you ever get the app to display the answer?

**Comment by RNetc0rp** [comment]
Nope, I ran out of credits for the GPT API (or so the Open AI customer support said) and moved onto trying to make gqml models run on my computer (to hopefully use it in place of GPT).
